src/elin_bjoernson_ex/ex04/myrand.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+import random
+
+__author__ = 'Elin Bjørnson'
+__email__ = 'elinbj@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+
+    def rand(self):
+        """ Returns the next random number"""
+        a = 7**5
+        m = 2**31-1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called